### ISSUE TITLE
electron count defaults to a value which bypasses the trigger

### ISSUE THREAD

**Issue by bryngemark** [LDMXAnalysis/ldmx-sw#1017]
**Describe the bug**
The improved electron-multiplicity aware trigger now exits if the electron count is < 1. This is in many ways sensible, but, the electron count is currently not always set to the actual number, which means `getElectronCount()` returns -1, which is the default set in `Event.h`

**To Reproduce**
Run the trigger processor in debug logger mode and behold the printout. 
``` python 
from LDMX.Recon.simpleTrigger import simpleTrigger
...
p.sequence  = [simpleTrigger]
p.termLogLevel = 0
```

One can then explicitly set the electron count to 1 and see the difference: inject 
``` python 

from LDMX.Recon.electronCounter import ElectronCounter
eCount = ElectronCounter( 1, "ElectronCounter") # first argument is number of electrons in simulation 
eCount.use_simulated_electron_number = True
p.sequence.extend( [eCount] )
```

**Desired behavior**
I think it would make sense for the electron count to default to 1.
